apps/web/app/datadog.ts

```diff
@@ -14,7 +15,7 @@ export default function DatadogInit() {
     if (isDevelopment) return;
 
     if (!nextPublicDatadogAppId || !nextPublicDatadogClientToken) {
-      console.warn('Datadog is not configured');
+      logger.warn('Datadog is not configured');
```

Comment:
will this actually end up anywhere? STDOUT?

Reply:
it will end up in stdout.